bin/varnishtest/tests/v00016.vtc

```diff
@@ -103,7 +103,7 @@ varnish v1 -errvcl {Symbol 'foo' has wrong type (instance)} {
 	}
 }
 
-varnish v1 -syntax 4.0 -errvcl {Undefined acl foo} {
+varnish v1 -syntax 4.0 -errvcl {Expression has type directors.shard, expected ACL} {
```

Comment:
Does anyone^Wany @Dridi see any issue with this change?
I understand that preserving support VCL 4.0 syntax is important to you, but do we _really_ need to keep the error messages identical? The complications would not seem justified to me.

Reply:
pow wow agreed that error messages are not part of the API and may be subject to improvement